ruby/provision.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# installation settings
+DB="fcc_provision" # the name of postgreSQL DB we need to provision, maybe $2
+
+export DEBIAN_FRONTEND=noninteractive
+echo "---------------------------------------------"
+echo "Running vagrant provisioning"
+echo "---------------------------------------------"
+
+echo "---------------------------------------------"
+echo "------- Updating package dependencies -------"
+echo "------------ And adding repos ---------------"
+echo "---------------------------------------------"
+sudo sh -c "echo deb http://ppa.launchpad.net/chris-lea/node.js/ubuntu trusty main  >> /etc/apt/sources.list.d/chris-lea-node_js-trusty.list"
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys C7917B12
```

Comment:
This kind of prints some stuff in console, in _red text_

```bash
==> default: Executing: gpg --ignore-time-conflict --no-options --no-default-keyring --homedir /tmp/tmp.iQhDy6kVdi --no-auto-check-trustdb --trust-model always --keyring /etc/apt/trusted.gpg --primary-keyring /etc/apt/trusted.gpg --keyserver keyserver.ubuntu.com --recv-keys C7917B12
==> default: gpg: 
==> default: requesting key C7917B12 from hkp server keyserver.ubuntu.com
==> default: gpg: 
==> default: key C7917B12: public key "Launchpad chrislea" imported
==> default: gpg: 
==> default: Total number processed: 1
==> default: gpg: 
==> default:               imported: 1
==> default:   (RSA: 1)
==> default: OK
```

Maybe we can pipe the output to `/dev/null` and pipe error to output with `2>&1`